src/PyCall.jl

```diff
@@ -897,7 +897,10 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
             throw(MethodError(show, (io, mime, o)))
         end
         Base.showable(::$mime, o::PyObject) =
-            !ispynull(o) && hasproperty(o, $method) && let meth = o.$method
+            !ispynull(o) &&
+            !pyisinstance(o, @pyglobalobj :PyType_Type) &&  # issue 816
```

Comment:
Can we use `PyCallable_Check` instead?

Reply:
`Issue816._repr_html_` in the OP is callable so I'm not sure if `PyCallable_Check` is enough. But maybe `PyMethod_Check` would work? I'll check.